pkg/cmd/get/token/cmd.go

```diff
@@ -45,6 +45,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 
 	cmd.Flags().StringVar(&o.outputFile, "output-file", "", "The generated resources will be copied in the specified file")
 	cmd.Flags().BoolVar(&o.useBootstrapToken, "use-bootstrap-token", false, "If set then the bootstrap token will used instead of a service account token")
+	cmd.Flags().BoolVar(&o.jsonOutput, "json", false, "If set output will be json format")
```

Comment:
This works, but what if later we want to add yaml output? I think it will be better
to have:

    --output=text (defalt)
    --output=json

This is how kubectl works, working like other tools in this space will make this
easier to use.